deepchecks/tabular/checks/methodology/__init__.py

```diff
@@ -35,3 +39,10 @@
     'ModelInferenceTime',
     'DatasetsSizeComparison',
 ]
+
+warnings.warn(
```

Comment:
How would we remember to remove this coming 0.8?

Reply:
We should have an issue for that on the 0.8 milestone